backends/jwt_remote.go

```diff
@@ -40,6 +42,10 @@ type Response struct {
 	Error string `json:"error"`
 }
 
+const (
+	whitelistMagicForAnyHost = "*"
```

Comment:
Haha, funny name. I'd call it `hostsWildcard` or something like that, but I kind of like it.